spec/betterc.dd

```diff
@@ -83,7 +83,7 @@ $(H2 $(LNAME2 retained, Retained Features))
     $(LI `assert` failures are directed to the C runtime library)
     )
 
-$(H2 $(LNAME2 consequences, Not Available))
+$(H2 $(LNAME2 consequences, Not Available Features))
```

Comment:
Doesn’t it sound better with “Features Not Available”?

Reply:
The section about is titled: "Retained Features"
https://dlang.org/spec/betterc.html#retained

Hence, I went with "Not Available Features", though I don't have a strong feeling about this as a non-native speaker.

Reply:
"Unavailable Features" would be much better than "Not Available Features," though "Features Not Available" works fine too.